src/components/menuItemPropTypes.js

```diff
@@ -154,6 +154,12 @@ const propTypes = {
     /** Should render component on the right */
     shouldShowRightComponent: PropTypes.bool,
 
+    /** Should render title without tooltip or not, we don't show tooltip on some items like, workspace and few more */
```

Comment:
This could be clearer, it's not obvious what `shouldUseFullTitle` should be set to in order to display a tooltip.